tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/Common/ScenarioContextKeys.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps.Common
+{
+    internal static class ScenarioContextKeys
+    {
+        public const string AccessTokenKey = "AccessToken";
```

Comment:
To key on the end or to no on the end? Just highlighting to be consistent with the namings of the constants within this class. Either remove 'key' from the end of AccessToken or add key to the end of OrganisationMapDictionary.